src/daedalus.cpp

```
@@ -452,63 +427,26 @@ class daedalus_ode {
             .sum(Eigen::array<Eigen::Index, 1>{1})
             .reshape(Eigen::array<Eigen::Index, 2>{n_strata, 1});
 
-    /// scale contacts in different settings, and sum for total contacts
-    // internal.scaling_factor is a placeholder
-    internal.cm_temp =
-        shared.cm.contract(internal.scaling_factor, vec_tensor_dims);
+    const size_t id_npi_regime =
```

Clang (14.0.6)

[warning] variable 'id_npi_regime' is not initialized

(cppcoreguidelines-init-variables)